index.html

```diff
@@ -37,15 +34,14 @@ <h2 class="john">I'm a Software Developer</h2>
             a project you need coded,
             don’t hesitate to contact me.
         </p>
-    </div>
-    <div class="SM-Container">
-        <div class="box"><img src="Icon\GitHub\Tertiary\Icons\Icon -GitHub.png" alt="github icon"></div>
-        <div class="box"><img src="Icon\LinkedIn\Tertiary\Icons\Icon - Linkedin.png." alt="linkedin icon"></div>
-        <div class="box"><img src="Icon\Twitter\Tertiary\Icons\Icon - Twitter.png" alt="twitter icon"></div>
-        <div class="box"><img src="Icon\Medium\Tertiary\Icons\Icon - Medium.png" alt="medium icon"></div>
-        <div class="box"><img src="Icon\Angelist\Tertiary\Icons\Icon - Angelist.png" alt="angelist icon"></div>
-    </div>
-    <div class="image"><img src="Images\Illustration.png" alt="background-image"></div>
+    <ul class="SM-Container">
+        <li class="box"><a href= "https://github.com/JohnteFresh"><img src="Icon\GitHub\Tertiary\Icons\Icon -GitHub.png" alt="github icon"></a></li>
+        <li class="box"><a href= "https://www.linkedin.com/in/johnson-irungu-85475a268/"><img src="Icon\LinkedIn\Tertiary\Icons\Icon - Linkedin.png." alt="linkedin icon"></a></li>
```

Comment:
- [ ] You did a great job adding the required changes @JohnteFresh, But while doing so I have noticed two of your social media icons are not visible on the webpage. This is because of how you apply the `src` property. I can see one of your image is followed by a `.` at the end. Please make sure all social media icons are  visible and resubmit the project, 
![image](https://github.com/JohnteFresh/Mobile-Portfolio-setup/assets/69077061/cdf774da-1751-4404-b987-c61feb3e83d8)

Reply:
Thank you @TadesseDev I hope my update passes.